benchmarks/README.md

```diff
@@ -80,6 +80,67 @@ using `pyarrow` dtypes).
 | Dependencies._update_media()                    |    0.087 |    0.086 |     0.145 |
 | Dependencies._update_media_version(10000 files) |    0.011 |    0.011 |     0.020 |
 
+## audb.Dependencies methods using polars
+
+Pyarrow was further  compared to a reimplenentation  of the dependency
+module  by  monkeypatching using  the  polars  dataframe engine.   All
+methods of `audb.Dependencies` were reimplemented using `polars`.
+
+* CPU: 12th Gen Intel Core i7-1255U
+* RAM: 15.66 GB
+* Hard Drive: KBG5AZNT1T02 LA KIOXIA
+* Linux: Ubuntu 22.04.4 LTS
+* Python 3.11.9
+
+To run the benchmark execute:
+
+```bash
+$ python benchmark-dependencies-methods-polars.py
+```
+
+The data were compared to
+the results from `benchmark-dependencies-methods.py` using
+
+```bash
+python compare_dependency_methods_polars.py
+```
+Both steps require that `benchmark-dependencies-methods.py`
+has been run previously in order to create the test data
+and results.
+
+| method                                          |   pandas |   polars | winner   |   factor |
+|-------------------------------------------------|----------|----------|----------|----------|
+| Dependencies.__call__()                         |    0.000 |    0.000 | polars   |    2.667 |
+| Dependencies.__contains__(10000 files)          |    0.003 |    0.002 | polars   |    2.005 |
+| Dependencies.__get_item__(10000 files)          |    0.648 |    0.013 | polars   |   50.382 |
+| Dependencies.__len__()                          |    0.000 |    0.000 | pandas   |    1.300 |
+| Dependencies.__str__()                          |    0.004 |    0.000 | polars   |   24.677 |
```

Comment:
To have a nicer rendering in markdown:

```suggestion
| Dependencies.\_\_call__()                         |    0.000 |    0.000 | polars   |    2.667 |
| Dependencies.\_\_contains__(10000 files)          |    0.003 |    0.002 | polars   |    2.005 |
| Dependencies.\_\_get_item__(10000 files)          |    0.648 |    0.013 | polars   |   50.382 |
| Dependencies.\_\_len__()                          |    0.000 |    0.000 | pandas   |    1.300 |
| Dependencies.\_\_str__()                          |    0.004 |    0.000 | polars   |   24.677 |
```

Reply:
Will this step always requires handcrafting escapes? Or can this be solved programmatically?

Reply:
I'm sure you can find a way to solve it programmatically, but I'm not sure if it is worth the effort. Maybe there exists an external package that can escape strings for markdown?